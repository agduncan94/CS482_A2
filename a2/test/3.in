>seq1
ANIDASNDNIAIDANIDAIDNASIODNSAIDSADINAS
>seq2
DNSAIDANIDANDAIND
