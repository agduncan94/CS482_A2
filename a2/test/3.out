-8
ANIDASNDNIAIDANIDAIDNASIODNSAIDSADINAS
-N---S----AIDANIDA--N----D--AI-----N--
